src/server/auth.js

```diff
@@ -97,7 +97,15 @@ function setupAuth(app) {
       tag: "login"
     });
 
-    const callbackUrl =`https://${req.get("host")}/do-login?${params.toString()}`;
+    let callbackUrl;
+
+    if (process.env.ENVRIONMENT === "production") {
```

Comment:
Can you change this section so there is less duplication? 

```
const protocol = process.env.ENVRIONMENT === "production" ? "https" : "http";
const callbackUrl = protocol + ...;
```

Reply:
I just stumbled over this PR from 2 years ago and noticed that there's a typo in `ENVRIONMENT`.